tests/killerRabbit.php

```diff
@@ -4,7 +4,7 @@
 
 require __DIR__ . '/../vendor/autoload.php';
 
-if (!isset($argv[1]) || (intval($argv[1]) != $argv[1])) {
```

Comment:
maybe we should keep the second condition here but just with `!==`?

Reply:
That's not possible, because `intval($argv[1])` returns `int` and plain `$argv[1]` is string.

Later converting to `(int)` must be enough

Reply:
oh yeah, of course:)